llama_index/output_parsers/selection.py

```diff
@@ -82,8 +82,13 @@ def _format_output(self, output: List[dict]) -> List[dict]:
         return output_json
 
     def parse(self, output: str) -> Any:
-        output = _marshal_llm_to_json(output)
-        json_output = json.loads(output)
+        json_output = _marshal_llm_to_json(output)
```

Comment:
Note: this also fixes a latent issue, where the returned `StructuredOutput` had a `raw_output` that was already marshalled away from raw